LeafBot/LeafBot/Commands/Games.cs

```diff
@@ -88,7 +89,26 @@ public async Task PickRole(CommandContext ctx)
           result += ", ";
         }
       }
+
+      // Add number of roles to stats
+      Stats.RolesPicked += numberOfRoles;
+
       await ctx.RespondAsync($"{ctx.Member.Mention}, you will queue for {result}. {Strings.COMMANDING_STRINGS[random.Next(Strings.COMMANDING_STRINGS.Length)]}!");
     }
+
+    [Command("bunnyage")]
+    [Description("Using advanced mathmatics, trigenometric reverse algebra and college level pagan science, Leafbot will work out your age in bunny years")]
+    [Aliases("age", "mebunny", "bunage")]
+    public async Task BunnyAge(CommandContext ctx, [RemainingText, Description("Your age in homan years")] int age = 0)
+    {
```

Comment:
suggestion: perhaps respond with a helper message if age wasn't entered? ((as opposed to saying you are 0.0 years old))

Reply:
Yeah good suggestion, originally I did have some snarky messages if you didn't enter an age correctly. I'll add those back in

Reply:
Added some snarky comments when no age is entered :D